src/containers/AssetList.js

```diff
@@ -38,15 +38,15 @@ class AssetList extends Component {
     const newQuery = this.updateQueryWithDept({
       ...query, ...(field === null ? DEFAULT_QUERY : {}), search
     });
-    this.getAssets(query, newQuery);
+    this.getAssetsIfChanged(query, newQuery);
```

Comment:
Could you update the PR description to outline why this is included in it? I know it's a bug-fix to another issue which you prefer but us-in-six-months might like a little pointer to the issue this is supposed to fix.

Reply:
This was a fix for a significant bug where the asset list didn't update when the filter changed. Unfortunately the exact steps to repeat this bug escape me.